src/main/java/software/amazon/awssdk/http/apache/internal/conn/SdkTlsSocketFactory.java

```diff
@@ -47,34 +49,32 @@ protected final void prepareSocket(final SSLSocket socket) {
         // BEGIN GG MODIFICATIONS
         try {
             SSLParameters params = new SSLParameters();
-            params.setApplicationProtocols(new String[]{"x-amzn-http-ca", "http/1.1"});
+            params.setApplicationProtocols(new String[] {"x-amzn-http-ca", "http/1.1"});
             socket.setSSLParameters(params);
         } catch (NoSuchMethodError e) {
             log.warn(() -> "Unable to configure socket for ALPN. Ports other than 443 may still work."
                     + " Your Java version is more than 3 years outdated, "
                     + "update it to a version newer than Java 8u252.");
-            // Java 8 did not launch with ALPN support, but it was backported in April 2020 with JDK8u252.
-            // Catching the error here so that we can continue to work with older JDKs since the user may not
+            // Java 8 did not launch with ALPN support, but it was backported in April 2020
+            // with JDK8u252.
+            // Catching the error here so that we can continue to work with older JDKs since
+            // the user may not
             // even require ALPN to work.
         }
         // END GG MODIFICATIONS
 
         log.debug(() -> String.format("socket.getSupportedProtocols(): %s, socket.getEnabledProtocols(): %s",
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Potential log injection detected. Ensure all untrusted input is properly sanitized before logging. Use parameterized logging or validate input against a allow list to prevent log injection vulnerabilities. Consider using a dedicated logging library&#39;s built-in sanitization features when available. Learn more - https://cwe.mitre.org/data/definitions/117.html